CONTRIBUTING.md

```diff
@@ -224,7 +224,7 @@ In case the commit is rejected because it's not formatted you can run
 the following to format the code:
 
 ```
-go fmt ./...
+goimports -w ./
```

Comment:
Do we need to add a line about installing `goimports`? Or does it come standard with `go` installations?

Reply:
Good point - added a commit to address this.